app/locales/en/translations.js

```diff
@@ -17,4 +17,56 @@ export default {
         pre_moderation: `pre-moderation`,
         post_moderation: `post-moderation`,
     },
+    setup: {
+        start: `Start Moderating`,
+        which: `Which provider would you like to set up?`,
```

Comment:
Missed this on the first pass. This should say "Which provider would you like to set up first?" (missing "first").